package.json

```diff
@@ -38,12 +38,15 @@
     "node": ">= 4.0.0"
   },
   "devDependencies": {
-    "tape": "~4.2.0",
+    "babel-eslint": "^8.2.5",
     "bindings": "~1.2.1",
+    "eslint": "^5.0.1",
     "nan": "^2.0.0",
-    "require-inject": "~1.3.0"
+    "require-inject": "~1.3.0",
+    "tape": "~4.2.0"
   },
   "scripts": {
-    "test": "tape test/test-*"
+    "lint": "node_modules/eslint/bin/eslint.js --no-eslintrc --rule no-unused-vars:error bin lib test",
```

Comment:
Can you not just do:

```diff
-     "lint": "node_modules/eslint/bin/eslint.js --no-eslintrc --rule no-unused-vars:error bin lib test", 
+     "lint": "eslint --no-eslintrc --rule no-unused-vars:error bin lib test", 
```

Also is there a reason to add the lint rules here instead of just adding a `.eslintrc.yaml`? I assume we're going to add more rules in the future...

Reply:
Had listed just the one rule for now, since didn't want to introduce tons of style changes all at once (and different parts of the codebase follow different standards...)

Reply:
Oh yeah, I didn't mean that you should add a full set of rules, just that adding a second rule will be a lot easier if it's just adding a line to a yaml file.

I think using `.eslintrc.yaml` is the convention in everything but nodejs/node (and that's a recent change).